test/services/user/registration.test.js

```diff
@@ -371,4 +371,37 @@ describe('registration service', () => {
 				});
 			});
 	});
+
+	it('hashService returns a string', () => {
+		const email = `max${Date.now()}@mustermann.de`;
+		const hashData = {
+			toHash: email,
+			save: true,
+			patchUser: true,
+		};
+
+		hashService
+			.create(hashData)
+			// eslint-disable-next-line promise/always-return
+			.then((res) => {
+				expect(res).to.be.a('string');
+			})
+			.catch(() => {});
+	});
+
+	it('hashService returns bad request without toHash parameter', () => {
+		const hashData = {
+			save: true,
+			patchUser: true,
+		};
+
+		hashService
+			.create(hashData)
+			// eslint-disable-next-line promise/always-return
+			.then(() => {})
```

Comment:
you should add something like `expect.fail('This should not happen')` here, otherwise the test will be green even if the method doesn't fail as expected

Reply:
added